addons/website_mass_mailing/static/src/js/website_mass_mailing.editor.js

```diff
@@ -42,75 +59,79 @@ var mass_mailing_common = options.Class.extend({
     },
 });
 
-options.registry.mailing_list_subscribe = mass_mailing_common.extend({
-    cleanForSave: function () {
-        this.$target.addClass('d-none');
-    },
-});
-
-options.registry.newsletter_popup = mass_mailing_common.extend({
+options.registry.newsletter_popup = options.registry.mailing_list_subscribe.extend({
     popup_template_id: "editor_new_mailing_list_subscribe_popup",
     popup_title: _t("Add a Newsletter Subscribe Popup"),
-    select_mailing_list: function (previewMode, value) {
+
+    /**
+     * @override
+     */
+    start: function () {
         var self = this;
-        return this._super(previewMode, value).then(function (mailing_list_id) {
-            ajax.jsonRpc('/web/dataset/call', 'call', {
-                model: 'mail.mass_mailing.list',
-                method: 'read',
-                args: [[parseInt(mailing_list_id)], ['popup_content'], self.options.recordInfo.context],
-            }).then(function (data) {
-                self.$target.find(".o_popup_content_dev").empty();
-                if (data && data[0].popup_content) {
-                    $(data[0].popup_content).appendTo(self.$target.find(".o_popup_content_dev"));
-                }
-            });
+        this.$target.find('.o_edit_popup').on('click.newsletter_popup_option', function (ev) {
```

Comment:
You can write
```javascript
this.$target.on('click.newsletter_popup_option', '.o_edit_popup', function (ev) {
```
it will be more efficient and more easy to `.off` in destroy